codalib/bagatom.py

```diff
@@ -202,7 +202,7 @@ def nodeToXML(nodeObject):
     Take a Django node object from our CODA store and make an XML
     representation
     """
-
+    nodeObject_status = {'0': 'Inactive', '1': 'Active'}
```

Comment:
I'd prefer we didn't hardcode this dictionary here, as then we have two places to change (in two different repos) if the original choices undergo some alteration. Instead, if we're just after the status wording, why not just use `nodeObject.get_status_display()` on line 222 as we did in the templates in coda?

Reply:
I don't think we can use it here because codalib is not a django project.

Reply:
codalib isn't a Django project, but the docstring indicates that this function is expecting to be called _from_ a Django project, so I think we could still use this, but I certainly could be wrong. Any thoughts on this, @ldko?

Reply:
For questions like this, I would try it out:
```
>           statusNode.text = nodeObject.get_status_display()
E           AttributeError: 'NodeStub' object has no attribute 'get_status_display'
```
and if I look at the members of `nodeObject` in the nodeeToXML I don't see that method:
```
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', 'last_checked', 'node_capacity', 'node_name', 'node_path', 'node_size', 'node_url', 'status']
```
So, I think Madhulika might be right.

Reply:
Okie dokie, I stand corrected. I withdraw this comment :smile: